### DISCUSSION TITLE
Punchy 0.1.0 Release Planning

### DISCUSSION THREAD

**Discussion by zicklag** [fishfight/punchy#252] [Planning]
Release Discussion for the [0.1.0 Release](https://github.com/fishfolks/punchy/milestone/6).

**Comment by zicklag** [comment]
For the last bit I've been busy working on re-designing my organization's website, and I expect that to occupy me for _probably_ another week before I can get to a good stopping point, but, as always, timing is a little hard to judge.

As soon as I can get that stable enough up I'm going to be coming back to my tasks on Punchy.

I'm hoping to integrate a scripted game element now that `bevy_mod_js` has browser support: #253.

**Reply by zicklag** [reply]
Other than awaiting review, I finished up #253 which finishes up our Scripting MVP!

I also added a section to our Architecture document and new Scripting page to give a quick overview of the scripting system.

That's probably all I'm taking on for this release, because I'm going to start work on getting Jumpy shipable now.

**Comment by odecay** [comment]
Wasn't this one going to be 0.1.0?

**Reply by zicklag** [reply]
I vaguely remember something about that, but I can't remember. 🤷‍♂️

**Reply by erlend-sh** [reply]
[Yep](https://github.com/fishfolks/punchy/discussions/234). Seems like it was needed too, because zicklag was calling this the 0.5.0 release, even though the previous one was 0.0.4, heh. It’s about time!
